### PR TITLE
doma: Add support for XS 8GB

### PR DESCRIPTION
**Pull request by rshym**
* Add support for XS 8GB
* Use generic.cfg as default for all boards

### CODE REVIEW

**Comment by otyshchenko1** [comment]
lgtm

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/doma.bb [hunk 1]
```diff
@@ -15,9 +15,8 @@ SRC_URI = "\
 
 S = "${WORKDIR}"
 
-DOMA_CONFIG_salvator-x-m3-xt = "doma-salvator-generic.cfg"
-DOMA_CONFIG_salvator-x-h3-xt = "doma-salvator-generic.cfg"
-DOMA_CONFIG_salvator-xs-h3-xt = "doma-salvator-generic.cfg"
+DOMA_CONFIG = "doma-salvator-generic.cfg"
+# override generic config for boards with 8GB
```
Comment (andr2000) [inline recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/doma.bb]: not sure this comment is really needed

**Comment by andr2000** [comment]
`Reviewed-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>`

**Comment by rshym** [comment]
Reviewers added.
@iartemenko could you please merge? Patches are reviewed and approved.

**Pull request merged**